app/models/food.rb

```diff
@@ -0,0 +1,7 @@
+class Food < ApplicationRecord
```

Comment:
I see you started building a model - remember the difference between a Model and a PORO is that a Model will have a corresponding database table, and we'll use ActiveRecord to query for that data. Whereas a PORO is a "Plain Old Ruby Object" that we can make out of data we retrieve from the API. 

So, you did make a Food PORO which is perfect! We can set that up like a Mod 1 object and have `attr_reader`s and `def initialize` etc. 

I suggest you keep going on this to get good practice - it feels like the ideas are there, but you may be shaky on how to apply them.